internal/server/v1/variables/golden/bulk_variables/california.json

```diff
@@ -43731,7 +43731,6 @@
         "dc/031mxslrxvjv8",
         "dc/03288qhhv2df4",
         "dc/033zp3qb7s7j6",
-        "dc/034hcwptwt6d5",
```

Comment:
Any ideas on this? https://datacommons.org/browser/dc/034hcwptwt6d5

Reply:
The old SV https://datacommons.org/browser/Count_Person_40To64Years_RatioToPovertyLineUpto1.38_HispanicOrLatino will be retained.